### PR TITLE
Add cronjob trigger

### LINKED ISSUE #353: cronjob trigger
**What would you like to be added**:
Is it possible to manual trigger job in cronjob view. Normally, I have to open a terminal to execute kubectl create job. 

**Why is this needed**:
If we can trigger job more easy. I think that is good for testing.

**Environment you are Lens application on:**
Dev, QA, beta, Stage, Production
- Kubernetes distribution: [EKS, AKS, GKE, Rancher, ...]
K8S, EKS
- Desktop OS: <the OS Lens app runs on in your case>
Mac Catalina

### PR DESCRIPTION
**Pull request by nox-404**
Hello,

* Add the ability to trigger a CronJob (Inspired by kubernetes dashboard trigger feature)
* Fix the display of the "Last Scheduled" saying `<1m` when the cronjob never ran (now shows a dash)
* Change the overview of cronjobs, now showing Scheduled instead of "Running" and "Suspended" in gray instead of "Failed"

I did not edit the translations besides the auto-generated English...

This shall fix #353 and #592

### CODE REVIEW

**Comment by nox-404** [comment]
image
image
image
image

**Comment by nox-404** [comment]
By the way I have this upon adding a new cluster:

image

I also get this on master.

Upon click on Ok, it continues without any further problem

File: src/renderer/components/+workloads-cronjobs/cronjob-trigger-dialog.tsx [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+          />
+        </div>
+      </>
+    )
+  }
+
+  render() {
+    const { className, ...dialogProps } = this.props;
+    const cronjobName = this.cronjob ? this.cronjob.getName() : "";
+    console.log(cronjobName);
```
Comment (nevalla) [inline src/renderer/components/+workloads-cronjobs/cronjob-trigger-dialog.tsx:100]: Some debugging leftover?

**Review by nevalla — APPROVED** [review]
Thanks! Left one minor comment about `console` call, otherwise looks good to me. @aleksfront @ixrock  PTAL too.

File: src/renderer/components/+workloads-cronjobs/cronjob-trigger-dialog.tsx [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+  }
+
+  close = () => {
+    CronJobTriggerDialog.close();
+  }
+
+  onOpen = async () => {
+    const { cronjob } = this;
+    this.jobName = cronjob ? cronjob.getName() + "-manual-" + Math.random().toString(36).slice(2, 7) : "";
+    this.jobName = this.jobName.slice(0, 63);
```
Comment (ixrock) [inline src/renderer/components/+workloads-cronjobs/cronjob-trigger-dialog.tsx:47]: Why `63`? Please extract to some variable/class-prop and re-use in `maxLength` (if necessary)?

**Review by ixrock — APPROVED** [review]
LGTM

**Comment by nevalla** [comment]
@Nox-404 Please could you resolve conflicts.

File: src/renderer/components/+workloads-cronjobs/cronjob-trigger-dialog.tsx (hunk 1 above)
Comment (nox-404) [reply src/renderer/components/+workloads-cronjobs/cronjob-trigger-dialog.tsx:100]: Woups shame on me, will fix it now

File: src/renderer/components/+workloads-cronjobs/cronjob-trigger-dialog.tsx (hunk 2 above)
Comment (nox-404) [reply src/renderer/components/+workloads-cronjobs/cronjob-trigger-dialog.tsx:47]: What about I add an `rfc1123Label` validator and for that slice just add a comment to the doc to where this restriction comes from ?

**Comment by nox-404** [comment]
> @Nox-404 Please could you resolve conflicts.

@nevalla  Of course, but can you tell me how you handle the translations ? do I just run `yarn extract:i18n` or is there a more subtle way ?

**Comment by nevalla** [comment]
> @nevalla Of course, but can you tell me how you handle the translations ? do I just run `yarn extract:i18n` or is there a more subtle way ?

I think you can do it on GitHub UI by clicking `Resolve conflicts` button and accept changes that are coming from the master branch. If you want I can try to do it too.

**Pull request merged**